web-client/src/views/Messages/MessageTable.tsx

```diff
@@ -142,19 +142,26 @@ export const MessageTable = connect<
             </div>
           </div>
         )}
-        <div className="overflow-x-auto" id={id}>
-          <div className="grid-row grid-gap">
-            {messageFilters.length > 0 && (
-              <div className="desktop:grid-col-8 tablet:grid-col-12 display-flex flex-align-center">
-                <TableFilters
-                  filters={messageFilters}
-                  onSelect={updateMessageFilterSequence}
-                ></TableFilters>
-              </div>
-            )}
-            {selectable && getCompleteAllButton()}
-          </div>
 
+        <div className="grid-row grid-gap">
+          {messageFilters.length > 0 && (
+            <div
+              className={
+                selectable
+                  ? 'desktop:grid-col-8 tablet:grid-col-12 display-flex flex-align-center' // To take into account the Complete All button
+                  : undefined
+              }
+            >
+              <TableFilters
+                filters={messageFilters}
+                onSelect={updateMessageFilterSequence}
+              ></TableFilters>
+            </div>
+          )}
+          {selectable && getCompleteAllButton()}
+        </div>
+
+        <div className="overflow-x-auto overflow-y-hidden" id={id}>
```

Comment:
This was wrapped around the wrong section before and made the horizontal scrollbar appear spuriously at time. The `overflow-y-hidden` hides the vertical scrollbar, which otherwise sometimes appears.